### PR TITLE
Update python_wrapper.cpp

### PR DESCRIPTION
**Pull request by DerThorsten**
enforcing f_style, blind fix, untested

### CODE REVIEW

File: src/python_wrapper.cpp [hunk 1]
```diff
@@ -18,7 +18,7 @@ namespace marching_cubes
 
     std::tuple<py::array, py::array, py::array> 
     marching_cubes_glue(
-        py::array_t<int> volume,
+        py::array_t<int, , py::array::f_style > volume,
```
Comment (k-dominik) [inline src/python_wrapper.cpp]:
```suggestion
        py::array_t<int , py::array::f_style > volume,
```

Comment (DerThorsten) [reply src/python_wrapper.cpp]: hubs =)

**Comment by k-dominik** [comment]
cool, that way one will get consistent results for C- and Fortran-order. (will add a test to your PR if you don't mind)

**Comment by DerThorsten** [comment]
the forcast flag it should automatically  convert the array to the expected one (https://pybind11.readthedocs.io/en/stable/advanced/pycpp/numpy.html) 
Feel free to add a test =)

**Comment by k-dominik** [comment]
thank you @DerThorsten, it works as expected. I chose not to update the test-date which was previously incorrectly interpreted as Fortran. Will bump the version to 0.3 once this is in.

**Comment by munrocket** [comment]
@k-dominik can you please add a shell for local build? I want to make a PR also but can't build a project.

**Comment by munrocket** [comment]
```

**Comment by k-dominik** [comment]
> @k-dominik can you please add a shell for local build? I want to make a PR also but can't build a project.

I'd recommend using conda debug to setup a local build. Otherwise you might run into problems with passing CI. This will take care of all the dependency versions for you.

**Pull request merged**